freeipa.spec.in

```diff
@@ -73,10 +73,15 @@
 %global selinux_policy_version 3.14.3-52
 %global slapi_nis_version 0.56.4
 %global python_ldap_version 3.1.0-1
+%if 0%{?rhel} < 9
 # python3-lib389
 # Fix for https://bugzilla.redhat.com/show_bug.cgi?id=1912822
 # sync_repl issue
 %global ds_version 1.4.3.16-11
```

Comment:
ds_version 1.4.3.16-12 is required for the fix for https://github.com/389ds/389-ds-base/issues/4609

Reply:
Updated.